tests/generic_config_updater/test_dhcp_relay.py

```diff
@@ -0,0 +1,236 @@
+    +===========+==================+===========+================+=============+=======================+
+    |       100 | 192.168.100.1/24 | Ethernet4 | tagged         | disabled    |                       |
+    +-----------+------------------+-----------+----------------+-------------+-----------------------+
+    |       200 | 192.168.200.1/24 | Ethernet4 | tagged         | disabled    |                       |
+    +-----------+------------------+-----------+----------------+-------------+-----------------------+
+    '''
+    cmds = []
+    logger.info("Add vlans, assign IPs, add member")
+
+    for vlan in vlan_intfs_list:
+        cmds.append('config vlan add {}'.format(vlan['vlan_id']))
+        cmds.append("config interface ip add Vlan{} {}".format(vlan['vlan_id'], vlan['ip'].upper()))
+        cmds.append('config vlan member add {} {}'.format(vlan['vlan_id'], rand_vlan_port))
+    
+    logger.info("Commands: {}".format(cmds))
+    duthost.shell_cmds(cmds=cmds)
+
+def create_test_dhcp_servers(duthost, vlan_intfs_list):
+    '''Generate 4 dhcp server for each vlan
+
+    This VLAN detail shows below
+    +-----------+------------------+-----------+----------------+-------------+-----------------------+
+    |   VLAN ID | IP Address       | Ports     | Port Tagging   | Proxy ARP   | DHCP Helper Address   |
+    +===========+==================+===========+================+=============+=======================+
+    |       100 | 192.168.100.1/24 | Ethernet4 | tagged         | disabled    | 192.0.100.1           |
+    |           |                  |           |                |             | 192.0.100.2           |
+    |           |                  |           |                |             | 192.0.100.3           |
+    |           |                  |           |                |             | 192.0.100.4           |
+    +-----------+------------------+-----------+----------------+-------------+-----------------------+
+    |       200 | 192.168.200.1/24 | Ethernet4 | tagged         | disabled    | 192.0.200.1           |
+    |           |                  |           |                |             | 192.0.200.2           |
+    |           |                  |           |                |             | 192.0.200.3           |
+    |           |                  |           |                |             | 192.0.200.4           |
+    +-----------+------------------+-----------+----------------+-------------+-----------------------+
+    '''
+    cmds = []
+
+    # Generate 4 dhcp servers for each new created vlan
+    for vlan in vlan_intfs_list:
+        for i in range(1, 5):
+            cmds.append('config vlan dhcp_relay add {} 192.0.{}.{}'.format(vlan['vlan_id'], vlan['vlan_id'], i))
+    
+    duthost.shell_cmds(cmds=cmds)
+    time.sleep(5)
+    pytest_assert(duthost.is_service_fully_started('dhcp_relay'), "dhcp_relay service is not running during setup")
```

Comment:
>pytest_assert [](http://example.com/codeflow?start=4&length=13)

Could you also verify that the config takes effect by checking the command line parameters of `dhcpmon` ?